### PR TITLE
6821-redirect-after-reporting

### PR DESCRIPTION
**Pull request by bthaile**
added history to reporting report component to be used by submit initial reporter to redirect user to reporting markets.

https://app.clubhouse.io/augur/story/6821/redirect-after-report-submission

### CODE REVIEW

File: src/modules/reporting/actions/submit-initial-report.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { augur } from 'services/augurjs'
-import noop from 'utils/noop'
+import { REPORTING_REPORTING } from 'modules/routes/constants/views'
+import makePath from 'modules/routes/helpers/make-path'
 import logError from 'utils/log-error'
 import { getPayoutNumerators } from 'modules/reporting/selectors/get-payout-numerators'
 
-export const submitInitialReport = (marketId, selectedOutcome, invalid, callback = logError) => (dispatch, getState) => {
+export const submitInitialReport = (marketId, selectedOutcome, invalid, history, callback = logError) => (dispatch, getState) => {
```
Comment (justinbarry) [inline src/modules/reporting/actions/submit-initial-report.js]: Redirect worked! Should the market still appear in the designated report list?

screen shot 2018-02-27 at 1 04 38 pm

Comment (bthaile) [reply src/modules/reporting/actions/submit-initial-report.js]: Right! Need to refrsh market status. Or should it happen automagically! Will reseach

Comment (justinbarry) [reply src/modules/reporting/actions/submit-initial-report.js]: @bthaile Keep it here or open another task?

Comment (bthaile) [reply src/modules/reporting/actions/submit-initial-report.js]: I’ll open another task

Comment (bthaile) [reply src/modules/reporting/actions/submit-initial-report.js]: Yea, issue might be with augur-node or UI isn’t handling the re-emit log correctly.

Comment (justinbarry) [reply src/modules/reporting/actions/submit-initial-report.js]: Sweet. Thanks!

**Pull request merged**